src/styles/pages/get-involved/donate.module.scss

```diff
@@ -0,0 +1,51 @@
+.sectionContainer {
+  display: flex;
+  padding-top: 32px;
+  background-color: var(--color-white);
+  font-family: var(--font-inter);
+  padding: var(--edge-spacer);
+}
+
+.donateBox {
+  margin-top: 16px;
+  background-color: var(--color-beige);
+  padding-top: 16px;
+  padding-bottom: 16px;
+  padding-left: 32px;
+  padding-right: 32px;
```

Comment:
another fyi, if you define padding like so:
```padding: 5px 10px;```
It implements 5px padding on the top and bottom and 10px to left and right.